Core/Core/Store/Store.swift

```diff
@@ -179,6 +180,13 @@ public class Store<U: UseCase>: NSObject, NSFetchedResultsControllerDelegate {
         }
     }
 
+    @objc
+    public func controllerWillChangeContent(_ controller: NSFetchedResultsController<NSFetchRequestResult>) {
+        if #available(iOSApplicationExtension 13.0, *) {
```

Comment:
Unfortunately, the "@objc" prevents any of the nicer availability attributes from working